example-ideal.txt

```diff
@@ -0,0 +1,26 @@
+// ASSUME PROGRAM STOPS WHEN ALL BLOCKS ARE GRAY
+// we also want to use && to combine things (but it wasn’t necessary in this program)
+// our key words are “free” and “blocked” (which also wasn’t necessary in the empty room)
+// comments will not actually be implemented
+
```

Comment:
This is a really nice and simple syntax! We began with a similar idea of wanting to use while loops, but then we wanted to make our language look as not-code-like as possible. But I think for anyone with at least a little programming experience would find this language really intuitive.